src/protos/io/defang/v1/fabric.proto

```diff
@@ -690,3 +693,25 @@ message WhoAmIResponse {
   SubscriptionTier tier = 5;
   bool training_opt_out = 6;
 }
+
+message EstimateRequest {
+  string provider = 1;
+  string region = 2;
+  bytes pulumi_preview = 3;
+}
+
+message EstimateLineItem {
+  string description = 1;
+  string unit = 2;
+  float quantity = 3;
+  float cost = 4;
+  string currency = 5;
```

Comment:
Use "money" type? https://github.com/googleapis/googleapis/blob/master/google/type/money.proto

Reply:
I will look into it

Reply:
I spent some time trying to do this today, but so far, I haven't figured out how to import this dependency and get `buf` to recognize it. 

You can see my attempt in e612c4a2